cdap-runtime-ext-dataproc/src/main/java/co/cask/cdap/runtime/spi/provisioner/dataproc/DataProcConf.java

```diff
@@ -150,6 +155,44 @@ public SSHPublicKey getPublicKey() {
     return publicKey;
   }
 
+  /**
+   * @return GoogleCredential for use with Compute
+   */
+  public GoogleCredential getCredential() throws IOException {
```

Comment:
Should these two methods be named differently as they return credential for different purpose.

Reply:
done